tensorflow_recommenders/tasks/ranking.py

```diff
@@ -34,7 +33,10 @@ class Ranking(tf.keras.layers.Layer, base.Task):
 
   This task helps with building ranker models. Usually, these will involve
   predicting signals such as clicks, cart additions, likes, ratings, and
-  purchases.
+  purchases. Accordingly, this task is less prescriptive than `tfrs.Rerieval`
```

Comment:
Typo: `tfrs.Rerieval` -> `tfrs.tasks.Retrieval`

Reply:
> Thank you! Could you have a look at my comments?

You've been very patient, yes will address those comments right now!